### PR TITLE
PortAudio: Update to v190600_20161030

### PR DESCRIPTION
**Pull request by ligfx**
I couldn't figure out what the current version is (maybe a development snapshot?), but it seems to be from before V1905.

### CODE REVIEW

**Comment by ligfx** [comment]
Looks like the `pr-freebsd-x64` build is failing because 1) an old version of PortAudio is installed and 2) something adds `/usr/include` to include directories at top-level.

I'm not sure what to do about this. CMake best practices say to not use `include_directories`, but that's a much bigger change than just updating PortAudio.

**Comment by Orphis** [comment]
Regarding the global include directories, I agree and I plan on fixing that later.

**Comment by Orphis** [comment]
Can't add an inline comment so here it is!
You should change the lines in the CMakeLists.txt file like this (add the BEFORE keyword) to make it build.
It's a customization, but it should be easy enough to identify whenever we upgrade it again.
```
TARGET_INCLUDE_DIRECTORIES(portaudio BEFORE PRIVATE ${PA_PRIVATE_INCLUDE_PATHS})
TARGET_INCLUDE_DIRECTORIES(portaudio BEFORE PUBLIC include)
TARGET_INCLUDE_DIRECTORIES(portaudio_static BEFORE PRIVATE ${PA_PRIVATE_INCLUDE_PATHS})
TARGET_INCLUDE_DIRECTORIES(portaudio_static BEFORE PUBLIC include)
```

File: Externals/portaudio/CMakeLists.txt [hunk 1]
```diff
@@ -344,15 +347,15 @@ ENDIF()
...
 TARGET_LINK_LIBRARIES(portaudio ${PA_LIBRARY_DEPENDENCIES})
 
 ADD_LIBRARY(portaudio_static STATIC ${PA_INCLUDES} ${PA_COMMON_INCLUDES} ${PA_SOURCES} ${PA_NON_UNICODE_SOURCES})
 SET_PROPERTY(TARGET portaudio_static APPEND_STRING PROPERTY COMPILE_DEFINITIONS ${PA_PRIVATE_COMPILE_DEFINITIONS})
-TARGET_INCLUDE_DIRECTORIES(portaudio_static PRIVATE ${PA_PRIVATE_INCLUDE_PATHS})
-TARGET_INCLUDE_DIRECTORIES(portaudio_static PUBLIC include)
-TARGET_LINK_LIBRARIES(portaudio_static ${PA_LIBRARY_DEPENDENCIES})
+TARGET_INCLUDE_DIRECTORIES(portaudio_static BEFORE PRIVATE ${PA_PRIVATE_INCLUDE_PATHS})
+TARGET_INCLUDE_DIRECTORIES(portaudio_static BEFORE PUBLIC include)
+TARGET_LINK_LIBRARIES(portaudio_static BEFORE ${PA_LIBRARY_DEPENDENCIES})
```
Comment (Orphis) [inline Externals/portaudio/CMakeLists.txt]: You don't need before here

Comment (ligfx) [inline Externals/portaudio/CMakeLists.txt]: Yep, could've sworn I removed that before pushing..

**Pull request merged**